Reply/app/src/main/java/com/example/reply/MainActivity.kt

```diff
@@ -3,41 +3,60 @@ package com.example.reply
 import android.os.Bundle
 import androidx.activity.ComponentActivity
 import androidx.activity.compose.setContent
-import androidx.compose.foundation.layout.fillMaxSize
-import androidx.compose.material3.MaterialTheme
-import androidx.compose.material3.Surface
-import androidx.compose.material3.Text
 import androidx.compose.runtime.Composable
-import androidx.compose.ui.Modifier
 import androidx.compose.ui.tooling.preview.Preview
+import androidx.lifecycle.flowWithLifecycle
+import androidx.lifecycle.lifecycleScope
+import androidx.window.layout.FoldingFeature
+import androidx.window.layout.WindowInfoTracker.Companion.getOrCreate
+import com.example.reply.ui.*
 import com.example.reply.ui.theme.ReplyTheme
+import kotlinx.coroutines.flow.*
 
 class MainActivity : ComponentActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
+
+        /**
+         * Flow of [DevicePosture] that emits every time there's a change in the windowLayoutInfo
+         */
+        val devicePosture = getOrCreate(this).windowLayoutInfo(this)
```

Comment:
Small nit: I think it'd be clearer if `WindowInfoTracker` was kept at the front here, the direct import is a bit confusing:
```suggestion
        val devicePosture = WindowInfoTracker.getOrCreate(this).windowLayoutInfo(this)
```